market_v9/src/ext.rs

```diff
@@ -0,0 +1,106 @@
+// Copyright 2019-2022 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use fvm_ipld_encoding::serde_bytes;
+use fvm_ipld_encoding::tuple::*;
+use fvm_shared::address::Address;
+use fvm_shared::bigint::bigint_ser;
+use fvm_shared::econ::TokenAmount;
+
+use fvm_shared::sector::StoragePower;
+use fvm_shared::smooth::FilterEstimate;
+
+pub mod account {
```

Comment:
These structures and constants are already part of their respective crates. For example, `account/AuthenticateMessageParams` is already defined and exposed in `account_v11/src/types.rs`. The `ext.rs` files were originally used by test code to avoid having interdependencies. Let's get rid of all the `ext.rs` files.

Reply:
fixed